coalib/misc/Constants.py

```diff
@@ -109,11 +109,11 @@
                                     'languages',
                                     'definitions')
 
-system_coafile = os.path.join(coalib_root, 'default_coafile')
+system_coafile = os.path.join(coalib_root, 'system_coafile')
 
 user_coafile = os.path.join(os.path.expanduser('~'), '.coarc')
 
-default_coafile = '.coafile'
+system_coafile = '.coafile'
```

Comment:
Ah sorry, I didn't see the corresponding issue. See https://github.com/coala/coala/pull/3393#issuecomment-269140598